tools/asset-sync/assets.Tests.ps1

```diff
@@ -0,0 +1,430 @@
+BeforeAll {
+  . $PSScriptRoot/assets.Tests.Helpers.ps1
+  . $PSScriptRoot/assets.ps1
+
+  # clean up the temporary test repo structures
+  $testPath = Get-TestFolder
+  if (Test-Path $testPath){
+    Remove-Item -Recurse -Force $testPath
+  }
+
+  # clean up local .assets folder
+  $location = Resolve-AssetStore-Location
+  if (Test-Path $location){
+    Remove-Item -Recurse -Force $location
+  }
+
+  Set-StrictMode -Version 3
```

Comment:
Is there a reason this isn't at the top?

Reply:
Yeah. `Invoke-Pester` actually grabs the script blocks and invokes them, so any settings/varaibles that are outside of the expected scope are entirely ignored.

For me, I wanted specific code within the tests themselves to to blow up with strict errors.

Reply:
Ahh yeah I remember struggling with that before.